src/datasets/load.py

```diff
@@ -783,7 +783,7 @@ def get_module(self) -> DatasetModule:
         hfh_dataset_info = HfApi(config.HF_ENDPOINT).dataset_info(
             self.name,
             revision=self.revision,
-            token=token,
+            token=token if token else "no-token",
```

Comment:
would passing `token=False` work instead?

Reply:
also don't hesitate to ping @SBrandeis / @coyotte508 / @Pierrci on those kind of PRs =)

Reply:
`"no-token"` won't trigger the type checker so I think it's better

Reply:
If HFH maintainers prefer `"no-token"`, then this is OK! ;)